Mathlib/MeasureTheory/Integral/Lebesgue.lean

```diff
@@ -1640,6 +1674,77 @@ theorem _root_.IsFiniteMeasure.lintegral_lt_top_of_bounded_to_ennreal {α : Type
   exact ENNReal.mul_lt_top ENNReal.coe_lt_top.ne μ_fin.measure_univ_lt_top.ne
 #align is_finite_measure.lintegral_lt_top_of_bounded_to_ennreal IsFiniteMeasure.lintegral_lt_top_of_bounded_to_ennreal
 
+lemma tendsto_of_lintegral_tendsto_of_monotone {α : Type*} {mα : MeasurableSpace α}
```

Comment:
Could you give a docstring for this lemma and for the similar lemmas in this PR? The assumptions are long enough that I feel they deserve it.